### PR TITLE
Frdvo/c01 aws01

### PR DESCRIPTION
**Pull request by frdvo**
# Description

Exercise (e.g. C01-E01): 

Student Name: Fernando Rolnik

> Create two EC2 instances, with Amazon Linux, one bastion host with public IP and other internal instance, without public IP. One key for each instance. Ssh both.

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-aws01/frdvo/README.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+}
+
+
+
+**Add rule to allow Port TCP 22 inbond connections**
+`aws ec2 authorize-security-group-ingress \`
+    `--group-name SSHOpen \`
+    `--protocol tcp \`
+    `--port 22 \`
+    `--cidr 0.0.0.0/0`
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws01/frdvo/README.md:27]: to keep things more secure, avoid having a rule opening a port to the world(unless your SG is being used by a public service). Can you think in a way to have this SG more secure?

File: classes/01class/exercises/c01-aws01/frdvo/README.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+**Create the VM**
+
+`aws ec2 run-instances --image-id resolve:ssm:/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2 --instance-type t2.micro --count 1 --key-name da --security-group-ids sg-063fc9a232e4e2a60 --associate-public-ip-address`
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws01/frdvo/README.md:31]: nice use of the ssm parameter store to get the latest AMI

File: classes/01class/exercises/c01-aws01/frdvo/README.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+https://aws.amazon.com/amazon-linux-2/
+4 package(s) needed for security, out of 8 available
+Run "sudo yum update" to apply all updates.
+[ec2-user@ip-172-31-36-240 ~]$ exit
+
+
+- Commands to create the second EC2 instance and any additional resource required:
+
+**Generate SSH Key for Internal Servers**
+`aws ec2 create-key-pair --key-name daint --output text > daint.pem`
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws01/frdvo/README.md:170]: you can use a key to connect to multiple instances. there is no need to have a 1:1 relationship between keys and EC2 instances.
In any case, its not a problem and it should be fine. just bring you additional effort to manage the keys.

File: classes/01class/exercises/c01-aws01/frdvo/README.md (hunk 1 above)
Comment (frdvo) [reply classes/01class/exercises/c01-aws01/frdvo/README.md]: Yes, you are right. I should restrict to my IP address only. If my IP change, I should update it.

File: classes/01class/exercises/c01-aws01/frdvo/README.md (hunk 3 above)
Comment (frdvo) [reply classes/01class/exercises/c01-aws01/frdvo/README.md:170]: I agree. I just watched the reference video. The instructor recommends one key for the jump box and another key for the internal instances (As you say, for the internal we can share the same key). If the jump box key gets compromised, the attacker can't access the internal instances.

**Pull request merged**